managed/ui/src/components/config/Security/certificates/AddCertificateForm.js

```diff
@@ -193,8 +222,50 @@ export default class AddCertificateForm extends Component {
     setFieldTouched('clientCertPath', false);
     setFieldTouched('clientKeyPath', false);
     setErrors(newErrors);
-    this.setState({tab: newTabKey});
-  }
+    this.setState({ tab: newTabKey });
+  };
+
+  handleOnBlur = (event) => {
+    this.setState({
```

Comment:
Is this intended ? I mean if user add something like '/opt' and click outside , suggestion won't show as you are clearing suggestion text , right ?

Reply:
I need to verify this, Local setup is having some issues, I will get back to you.

Reply:
@gaurav061 : any update on this ? Other changes looks good to me .

Reply:
@sunilsutarDCE, yes It is the case when on blur we are hiding the text i.e if nothing is entered placeholder text will take its place and if something else is there we just want the user-entered value.